cmd/internal/wizard/wizard.go

```diff
@@ -143,7 +143,10 @@ func PromptForAPIKey() (string, error) {
 
 // PromptForEncryptionPassphrase handles user input for an encryption passphrase to be used with wizards
 func PromptForEncryptionPassphrase() (string, error) {
-	_, err := fmt.Print("Enter your encryption passphrase: ")
+	_, err := fmt.Print(`
+Data is encrypted on the network, with an encryption passphrase
```

Comment:
It looks like an extra empty line is printed. This line should start at the end of the previous one.

Reply:
The CLI seemed weird to me without a line break but your way is more authentic to the original mockup.

Reply:
Done